test/src/sml_boolean_test.c

```diff
@@ -22,7 +22,7 @@
 
 TEST_GROUP(sml_boolean);
 
-sml_buffer *buf;
+extern sml_buffer *buf;
```

Comment:
I'd mark them 'static' as I think the intention was not to share them between compilation units, or?

Reply:
well, the intention and implementation where diverging.
i have some slight preference for fixing without changing behaviour.
also, the local copies waste memory ;)
and it's proven to work with shared.